google-cloud-storage/lib/google/cloud/storage/file/list.rb

```diff
@@ -162,7 +163,7 @@ def all request_limit: nil, &block
           # @private New File::List from a Google API Client
           # Google::Apis::StorageV1::Objects object.
           def self.from_gapi gapi_list, service, bucket = nil, prefix = nil,
-                             delimiter = nil, max = nil, versions = nil,
+                             delimiter = nil, match_glob = nil, max = nil, versions = nil,
```

Comment:
I would add it to the end of the positional argument list, or even add it as a keyword argument, just so it doesn't create a "breaking change" to the interface. I know this is an internal, undocumented interface (marked `@private`) so we don't strictly have to maintain backward compatibility for it, but we might as well do so if we can. It will also reduce the chance of regressions because we missed a usage somewhere.

Reply:
I updated them all. 
Changing min to Ruby v2.7 for the gem as current supported version. Conformance test had a similar issue (min 2.7 req for `utils.rb`): setting to 3.2 as it runs in CI.